WordPress/src/main/java/org/wordpress/android/modules/ViewModelSubComponent.java

```diff
@@ -0,0 +1,17 @@
+package org.wordpress.android.modules;
+
+import org.wordpress.android.viewmodel.PluginBrowserViewModel;
+import org.wordpress.android.ui.stats.models.PostViewsModel;
+
+import dagger.Subcomponent;
+
+@Subcomponent
+public interface ViewModelSubComponent {
+
+    PluginBrowserViewModel pluginBrowserVM();
```

Comment:
Nitpick: I believe we generally prefer not to shorten things like `ViewModel` to `VM` especially in methods/properties. I'd have gone with `pluginBrowserViewModel`, but it's not a big deal :)

Reply:
fixed